IdentityCore/src/configuration/webview/MSIDWebviewConfiguration.h

```diff
@@ -50,6 +50,12 @@
 // PKCE Support
 @property (readonly) MSIDPkce *pkce;
 
+// State verification
+// Set this to YES to have the request fail at state verification failure.
+// Set this to NO if request should continue at state verification failure.
+// By default, this is set to YES.
+@property (readonly) BOOL stopAtStateVerificationFail;
```

Comment:
nit: can we call it something like "ignoreInvalidState"?
Also I didn't find it being set to YES anywhere. How can the default be YES?

Reply:
Woops, forgot to push. Anyways, changing it as suggested :)

Reply:
It was left in my local copy, will change with suggested name and push :)